docs/TextColumns.md

```diff
@@ -0,0 +1,77 @@
+_New in [:octicons-tag-24: 2.7.7](https://github.com/py-pdf/fpdf2/blob/master/CHANGELOG.md)_
+
+**Notice:** As of fpdf2 release 2.7.7, this is an experimental feature. Both the API and the functionality may change before it is finalized, without prior notice.
+
+
+## Text Columns ##
+
+The `FPDF.text_columns()` method allows to create columnar layouts, with one or several columns. Columns will always be of equal width.
+
+Beyond the parameters common to all text regions, the following are available for text columns:
+
+* l_margin (float, optional) - override the current left page margin.
+* r_margin (float, optional) - override the current right page margin.
+
+Only for `FPDF.text_columns()`:
+
+* ncols (float, optional) - the number of columns to generate (Default: 2).
+* gutter (float, optional) - the space required between each two columns (Default 10).
+
+
+#### Single-Column Example ####
+
+In this example an inserted paragraph is used in order to format its content with justified alignment, while the rest of the text uses the default left alignment.
+
+```python
+    cols = pdf.text_columns()
```

Comment:
In the documentation pages I wrote, I tried to always provide "autonomous" code snippets that could be copy & pasted and work "out of the box".

That meant always starting example code snippets with those lines:
```python
from fpdf import FPDF

pdf = FPDF()
pdf.add_page()
pdf.set_font("Times", size=36)
```

I wonder if we should try to stick to this philosophy in our documentation pages.
Code snippets would become a bit longer, but overally I think our documentation may be come more user-friendly.
What do you think @gmischler?

Reply:
Sounds like a good idea.